### ISSUE TITLE
Conduct an initial accessibility analysis of Libero Editor

### ISSUE THREAD

**Issue by NuclearRedeye** [libero/editor#469]
Development on Libero Editor has been progressing well, but the team haven't really been concentrating on ensuring the tool is capable of meeting accessibility requirements, and nor are the team sure what the accessibility requirements actually are. As such, we need somebody with experience to look at the tool and to define what we should be doing, and recommendations on how we can do it.

# DoD
- [ ] Accessibility Epic updated with a set of DoD criteria that the Editor _should_ be capable of meeting.
- [ ] A set of current issues identified and created with some recommendations or advice on what is needed to resolve them and measure those changes against the previous criteria.

**Comment by NuclearRedeye** [comment]
@discodavey I've fleshed out an issue to track your investigation work and set out some DoD criteria, let me know if there is anything that needs changing or that I missed (or is maybe out of scope for this issue) and if you still think the 0.5d estimate holds.

**Comment by discodavey** [comment]
I think 0.5d is fine cause of how small the editor is at the moment but if you want to be cautious then  do it as 0.75d. That should cover it for definite.

**Comment by discodavey** [comment]
I cannot edit your initial comments so adding the URL here:

https://libero-editor--staging--test-cluster.elifesciences.org/?articleId=54296

**Comment by discodavey** [comment]
I have done an initial test of the editor page above and have come across 18 issues to start off with. If you need me to put more detail in then please let me know.

https://docs.google.com/spreadsheets/d/19auNqSRagDSY3VK9LRrNXdaNpl_cB0_L9JhB4c32Zew/edit?usp=sharing

Thanks

**Comment by discodavey** [comment]
I have added the tickets on the epic

https://app.zenhub.com/workspaces/libero-editor-5d56a6cfaa13c900012ad75b/issues/libero/editor/489

There are a few that are missing as I did not understand what I have written. Once I have worked out what I mean't I will write those tickets
